sass/base/components/email-signup/_email-signup.scss

```diff
@@ -753,3 +755,56 @@ select,
     }
   }
 }
+
+// Privacy policy
+.cr-email-signup-fundraise-form,
+.cr-email-signup-register-interest-form,
+.cr-email-signup-workplace-form,
+.cr-email-signup-form {
+
+  .esu-privacy-message p,
+  .esu-initial-message p {
+    color: $colour-white;
+  }
+
+  &.block--cr-email-signup--step-2, 
+  &.block--cr-email-signup--step-3 {
+    .esu-privacy-message {
+      display: none;
+    }
+  }
+}
+
+.cr-email-signup-fundraise-form {
+  @include breakpoint($screen-lg) {
+    &.block--cr-email-signup.block--cr-email-signup--standard .block--cr-email-signup__inner-wrapper form {
+      position: absolute;
+      top: 35%;
```

Comment:
no need for position absolute there as the form already has been given position absolute earlier.

Reply:
Yeah :+1: true Done